lib/twitter_cldr/js/renderers.rb

```diff
@@ -18,6 +18,15 @@ module Calendars
 
       module Numbers
         autoload :NumbersRenderer,                      'twitter_cldr/js/renderers/numbers/numbers_renderer'
+        autoload :RBNFRenderer,                         'twitter_cldr/js/renderers/numbers/rbnf/rbnf_renderer'
```

Comment:
Usually it's recommended to keep your directories layout in sync with modules nesting, so if you have a directory `rbnf` (which is good), you should also have a module called `RBNF` that includes all the classes/modules defined within this directory. So, instead of `Numbers::RBNFRenderer` you would have `Numbers::RBNF::Renderer`. Then here within `Numbers` module you can autoload just the `RBNF` module and then autoload all nested classes in `rbnf.rb` file.